### PR TITLE
Bookstore project: Reducers and actions

### PR DESCRIPTION
**Pull request by Strangeal**
### Add reducers and actions
So far it has been enjoyable😊 implementing all the necessary requirements for the project I learned so many things and it was also fun working on this project, what I did in the whole project :

```...
/src
    |---/components
    |---/redux
        |--- /books
                | books.js
        |--- /categories
                | categories.js
        | configureStore.js
    | index.js
```
- Added action types for adding, checking status, and removing a book
- Added a reducer function for state changes for the actions created
- Configured the Redux Store in `configureStore.js`
- Imported the necessary methods from `Redux Toolkit`
- Combined both reducers into a root reducer by using the `configureStore` function.

### CODE REVIEW

File: src/components/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
```
Comment (TedLivist) [inline src/components/redux/books/books.js]: - [x] Kindly use the ducks pattern for your action type values.

  ```
  const ADD_BOOK = 'bookstore/books/ADD_BOOK';
  const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
  ```

  You can look at [this article](https://medium.com/@matthew.holman/what-is-redux-ducks-46bcb1ad04b7) for more information

File: src/components/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (TedLivist) [inline src/components/redux/categories/categories.js]: - [x] Kindly use the ducks pattern for your action type values.

  ```
  const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
  ```

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Strangeal,

Good job so far!
✔️ Good documentation
✔️ Implementation of reducers
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] You have not implemented the correct ducks' pattern for your files. The `redux` directory should be in the `src` folder, not the `components` folder. Kindly fix that.

  awwe

- [x] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @Strangeal 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**